packages/wrangler/src/package-manager.ts

```diff
@@ -0,0 +1,88 @@
+import { existsSync } from "node:fs";
+import { join } from "node:path";
+import { execa, execaCommandSync } from "execa";
+
+export interface PackageManager {
+  addDevDeps(...packages: string[]): Promise<void>;
+  install(): Promise<void>;
+}
+
+export async function getPackageManager(root: string): Promise<PackageManager> {
```

Comment:
100% bet you're going to get people asking for pnpm once they realise we're offering this option. We can do it later of course, but it's going to happen. (especially with folks slowly moving away from yarn)

Reply:
🤫